### PR TITLE
Add Submenu to example apps

### PR DESCRIPTION
**Pull request by reidbarber**
## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

**Review by snowystinger — APPROVED** [review]

File: examples/rsp-next-ts/pages/index.tsx [hunk 1]
```diff
@@ -162,6 +163,21 @@ export default function Home() {
                 <Item key="copy">Copy</Item>
                 <Item key="paste">Paste</Item>
                 <Item key="replace">Replace</Item>
+                <SubmenuTrigger>
```
Comment (yihuiliao) [inline examples/rsp-next-ts/pages/index.tsx:166]: Screenshot 2024-02-05 at 4 41 48 PM

for some reason i can't see the chevron's in the next.js app? seems fine in the cra one tho

Comment (reidbarber) [reply examples/rsp-next-ts/pages/index.tsx:166]: Weird, and they look fine in the collapsible table.

Comment (reidbarber) [reply examples/rsp-next-ts/pages/index.tsx:166]: Maybe we can merge the example apps and investigate in a follow-up

**Review by yihuiliao — APPROVED** [review]

File: examples/rsp-next-ts/pages/index.tsx (hunk 1 above)
Comment (reidbarber) [reply examples/rsp-next-ts/pages/index.tsx:166]: Looks like it was just due to the Next.js app having `box-sizing: border-box;` in its styles

**Pull request merged**